JobConfig/beam/prolog.fcl

```diff
@@ -217,7 +218,9 @@ mu2e.physics.g4s4Muons.producers.generate: {
    module_type: StoppedParticleReactionGun
    verbosityLevel : 1
    physics: @nil
-   muonStops : @local::mu2e.tgtMuonStops
+   muonStops : { @table::mu2e.tgtMuonStops
+   	       	 inputFiles : @local::mergedMuon_tgtStops_mdc2018
```

Comment:
This replaces @nil with a hardcoded set of input files.   I think this change is outside the scope of this PR, please take it out.

Reply:
Done (commit 3e76dc9e4)